packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl

```diff
@@ -4,6 +4,6 @@ verificationReminderSecond-title-3 = Don’t miss out on { -brand-mozilla }!
 verificationReminderSecond-description-3 = A few days ago you created a { -product-firefox-account }, but never confirmed it. Please confirm your account in the next 10 days or it will be automatically deleted.
 verificationReminderSecond-description-4 = A few days ago you created a { -product-mozilla-account }, but never confirmed it. Please confirm your account in the next 10 days or it will be automatically deleted.
 verificationReminderSecond-second-description = Your { -product-firefox-account } lets you sync your info across devices and unlocks access to more privacy-protecting products from { -brand-mozilla }.
-verificationReminderSecond-second-description-2 = Your { -product-mozilla-account } lets you sync your info across devices and unlocks access to more privacy-protecting products from { -brand-mozilla }.
```

Comment:
@bcolsson this string was modified/added after your review. `verificationReminderSecond-second-description-2` has never been used in product, and an update was request by UXC. Just checking that this looks good!

Reply:
It's deleted now, but no issues with this string. 👍